v23.1/cockroach-debug-encryption-decrypt.md

```diff
@@ -0,0 +1,63 @@
+---
+title: cockroach debug encryption-decrypt
+summary: Learn the commands for collecting debug information from all nodes in your cluster.
+toc: true
+docs_area: reference.cli
+---
+
+{{site.data.alerts.callout_danger}}
+We strongly recommend only using `cockroach debug encryption-decrypt` when working directly with the [Cockroach Labs support team](support-resources.html).
+{{site.data.alerts.end}}
+
+The [`cockroach debug encryption-decrypt`](cockroach-commands.html) command connects to your cluster and decrypts store files that are [encrypted at rest](encryption.html) so that they can be examined to add more context to logs gathered from the [`cockroach debug zip`](cockroach-debug-zip.html) command. Unless they are decrypted, Cockroach Labs cannot examine store files that are encrypted at rest.
+
+## Synopsis
+
+~~~ shell
+$ cockroach debug encryption-decrypt --enterprise-encryption={encryption_spec} --url=<connection string> {flags} {store_directory} {input_file} {output_file}
+~~~
+
+Replace:
+
+- `{encryption_spec}`: The cluster's [encryption details](encryption.html#starting-a-node-with-encryption).
+- `{store_directory}`: The directory where the node stores data files.
+- `{input_file}`: The name of an encrypted store file.
+- `{output_file}`: A name for the decrypted store file. If not specified, the command will output the decrypted contents to
+`STDOUT`.
+
+After running the command, you can send the requested decrypted file to Cockroach Labs.
```

Comment:
Worth putting another caveat here that you should only send if you're ok with CRL seeing the plaintext (i.e. unencrypted) contents of the file.

We have a warning at the top of the file, so maybe that's enough.

Reply:
SGTM.